modules/developers-guide/pages/cli-reference/dfx-build.adoc

```diff
@@ -23,7 +23,7 @@ You can use the following optional flags with the `+dfx build+` command.
 |===
 |Flag |Description
 
-|`+--check+` |Builds canisters using a temporary, hard-coded, locally-defined canister identifier for testing that your program compiles without connecting to the {IC} network.
+|`+--check+` |Builds canisters using a temporary, hard-coded, locally-defined canister identifier for testing that your program compiles without connecting to the {platform}.
 
 |`+-h+`, `+--help+` |Displays usage information.
 
```

Comment:
There is only one Internet Computer, therefore, in line 42, you probably want to mention that you can either deploy to the local canister execution environment (what <network> do we need to specify for that?) or the Internet Computer blockchain `ic`. External readers can't use testnets, so specifying additional environments will only confuse them...